### PR TITLE
Sort submitters

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Everything should be fixed now. Does isort work on your computer? Isort does not work on my machine... Can jou tell me, what diff you have, if you run `isort --recursive openslides`?

**Comment by emanuelschuetze** [comment]
Looks good now but wait with merge.

File: openslides/agenda/views.py [hunk 1]
```diff
@@ -286,7 +286,7 @@ def sort_speakers(self, request, pk=None):
                 raise ValidationError(
                     {'detail': _('Invalid data.')})
             valid_speakers.append(speakers[speaker_id])
-        weight = 0
+        weight = 1
```
[the remark this answers is not in the document]
Comment (FinnStutzenstein) [reply openslides/agenda/views.py]: Interesting question. I honestly do not know; I changed it back.

**Comment by emanuelschuetze** [comment]
Ok, please rebase.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Rebased.

**Comment by emanuelschuetze** [comment]
Ok, all done. Thanks.

**Pull request merged**